apps/frontend/src/pages/HomePage/HomePage.tsx

```diff
@@ -1,97 +1,74 @@
-  return (
-    <>
-      <section>
-        <h1>My Neighborhoods</h1>
-        <div className="all-neighborhoods">
-          <figure className="card">
-            <img src={neighborhoodImg1} alt="neighborhood" />
-            <figcaption>
-              <span className="info">
-                <h2>
-                  Palm Springs <span>(Admin)</span>
-                </h2>
-                <p>You can join other neighborhoods!</p>
-              </span>
-              <button className="explore-neighborhood">Explore</button>
-            </figcaption>
-          </figure>
+import { Container, Row, Col } from 'react-bootstrap';
+import NeighborhoodCard from '../../components/NeighborhoodCard/NeighborhoodCard';
 
-          <figure className="card">
-            <img src={neighborhoodImg2} alt="neighborhood" />
-            <figcaption>
-              <span className="info">
-                <h2>
-                  Up North <span>(Member)</span>
-                </h2>
-                <p>You can join other neighborhoods!</p>
-              </span>
-              <button className="explore-neighborhood">Explore</button>
-            </figcaption>
-          </figure>
-        </div>
-      </section>
+import styles from './HomePage.module.css';
+import { getStoredUser } from '../../utils/auth';
 
-      <section>
-        <h1>My active requests</h1>
-        <div className="all-active-requests">
-          <div className="request-card">
-            <img src="images/image.jpeg" alt="active request in neighborhood app" />
+import userServices from '../../services/users';
+import Request from '../../components/Request/Request';
 
-            <div className="request-card-content">
-              <p>Help! My cat is drowning</p>
-              <p>Created 11 Mar 2022 in Palm Springs Neighborhood</p>
-              <p>2 responses</p>
-            </div>
-          </div>
+// const neighborhoodImg1 = require('./images/palm-tree.jpeg');
+// const neighborhoodImg2 = require('./images/up-north.jpg');
 
-          <div className="request-card">
-            <img src="images/image.jpeg" alt="active requests on neighborhood app" />
+export async function loader() {
+  const user = getStoredUser();
+  if (!user) return null;
+  const userData = await userServices.getUserData(user.id);
+  return userData;
+}
 
-            <div className="request-card-content">
-              <p>Help with acute lazyness</p>
-              <p>Created 12 Mar 2022 in Up North Neighborhood</p>
-              <p>No responses</p>
-            </div>
-          </div>
-        </div>
-      </section>
+export default function HomePage() {
+  const userData = useLoaderData() as unknown as UserWithRelatedData;
+  const { neighborhoods } = userData;
+  const neighborhoodCards =
+    neighborhoods.length === 0 ? (
+      <p>You haven't joined any neighborhoods yet!</p>
+    ) : (
+      neighborhoods.map((neighborhood) => (
+        <Col className="pe-0" sm="6" md="4" lg="3">
```

Comment:
We get this error again that requires you to add unique keys for the `Col`s as well.

Reply:
I keep telling myself I will add the key and then I forget :).
Added it